modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -8,7 +8,7 @@ The Scheduler component enables you to trigger a flow when a time-based conditio
 Schedulers follow the timezone used by the machine where the Mule runtime is running. If an application is running in CloudHub, the timezone that the Scheduler conforms to corresponds to the region in which the CloudHub worker is running.
 
 // TODO: MULE-14438
-Only one flow triggered by the scheduler can be active at any given time.
+The scheduler component can trigger only one flow at any given time, and if a triggered flow is executing, it won't be triggered again until it finishes. The scheduler can not trigger multiple flows at same time.
```

Comment:
I would remove the sentence "The scheduler can not trigger multiple flows at same time."